src/CTA.WebForms2Blazor/Helpers/ControlHelpers/EmbeddedCodeReplacers.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace CTA.WebForms2Blazor.Helpers.ControlHelpers
+{
+    public static class EmbeddedCodeReplacers
+    {
+        //DataBind regex might have issue of taking front parentheses
+        public static Regex DataBindRegex = new Regex("<%#\\W*(?<expr>[^%>]*)\\s*%>");
+        public static Regex SingleExpRegex = new Regex("<%:\\s*(?<expr>[^%>]*)\\s*%>");
+        public static Regex DirectiveRegex = new Regex("<%@\\s*(?<expr>[^%>]*)\\s*%>");
+        public static Regex AspExpRegex = new Regex("<%\\$\\s*(?<expr>[^%>]*)\\s*%>");
+        
+        public static string ReplaceDataBind(Match m)
+        {
+            var expr = m.Groups["expr"].Value;
+            var newValue = "@(" + expr + ")";
+            return newValue;
+        }
+        
+        public static string ReplaceSingleExpr(Match m)
+        {
+            var expr = m.Groups["expr"].Value;
+            var newValue = "@" + expr;
```

Comment:
I think you should use explicit expression embedding with `@()` here, implicit embedding with just`@` is going to break if expressions with type parameters or spaces are present